src/plugins/dashboard/public/dashboard_container/embeddable/create/create_dashboard.test.ts

```diff
@@ -450,7 +451,9 @@ test('creates a control group from the control group factory and waits for it to
     'control_group'
   );
   expect(mockControlGroupFactory.create).toHaveBeenCalledWith(
-    expect.objectContaining({ controlStyle: 'twoLine' })
+    expect.objectContaining({ controlStyle: 'twoLine' }),
+    undefined,
+    { lastSavedInput: expect.objectContaining({ controlStyle: 'oneLine' }) }
```

Comment:
Refer to https://github.com/elastic/kibana/pull/175146/files#r1470120760 for why the `controlStyle` is `oneLine` in the `lastSavedInput`